### PR TITLE
Url shortener module

### PR DESCRIPTION
**Pull request by andela-ijubril**

### CODE REVIEW

**Comment by andela-ijubril** [comment]
@swapagarwal kindly help me add a `module` and `work in progress` to this PR

**Comment by andela-ijubril** [comment]
@swapagarwal kindly help me to add `module` and `work in progress` label to this PR

File: modules/src/url.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+from templates.text import TextTemplate
+
+
+GOOGLE_URL_SHORTENER = os.environ.get('GOOGLE_URL_SHORTENER', config.GOOGLE_URL_SHORTENER)
+
+
+def process(input, entities=None):
+    output = {}
+    try:
+        long_url = ""
```
Comment (williamshoops96) [inline modules/src/url.py]: Shouldn't you have `long_url = entities['url'][0]['value']` ?

Comment (andela-ijubril) [inline modules/src/url.py]: Thanks... Updated

**Comment by andela-ijubril** [comment]
Thanks @swapagarwal for adding the label and for the awesome work you doing with JARVIS-on-Messenger. I guess the next step for this is to configure it with wit.ai?

**Comment by swapagarwal** [comment]
Thanks a lot @andela-ijubril for your kind words! :smile: 
I'm just about to set it up on wit.ai and register with the Google API.

**Comment by swapagarwal** [comment]
Thanks @andela-ijubril! :+1:

**Comment by andela-ijubril** [comment]
Many Thanks, @swapagarwal 👍

**Pull request merged**